src/software/simulated_tests/simulated_test_fixture.cpp

```diff
@@ -57,6 +61,7 @@ void SimulatedTestFixture::SetUp()
     {
         enableVisualizer();
     }
+    setupReplayLogging();
```

Comment:
what happens when u run this locally? Could it use up a lot of storage over time?

Reply:
Disk use is pretty insignificant, looks to be around 2 MB total for running all of the simulated tests.

Reply:
Ah okay, we can clean up the cache later if its piling up somehow, not sure how bazel cleans its test artifacts